open_spiel/bots/stockfish/CMakeLists.txt

```diff
@@ -0,0 +1,69 @@
+add_library(stockfish OBJECT
+        stockfish/src/benchmark.cpp
+        stockfish/src/bitbase.cpp
+        stockfish/src/bitboard.cpp
+        stockfish/src/bitboard.h
+        stockfish/src/endgame.cpp
+        stockfish/src/endgame.h
+        stockfish/src/evaluate.cpp
+        stockfish/src/evaluate.h
+        stockfish/src/material.cpp
+        stockfish/src/material.h
+        stockfish/src/misc.cpp
+        stockfish/src/misc.h
+        stockfish/src/movegen.cpp
+        stockfish/src/movegen.h
+        stockfish/src/movepick.cpp
+        stockfish/src/movepick.h
+        stockfish/src/pawns.cpp
+        stockfish/src/pawns.h
+        stockfish/src/position.cpp
+        stockfish/src/position.h
+        stockfish/src/psqt.cpp
+        stockfish/src/search.cpp
+        stockfish/src/search.h
+        stockfish/src/thread.cpp
+        stockfish/src/thread.h
+        stockfish/src/thread_win32_osx.h
+        stockfish/src/timeman.cpp
+        stockfish/src/timeman.h
+        stockfish/src/tt.cpp
+        stockfish/src/tt.h
+        stockfish/src/tune.cpp
+        stockfish/src/tune.h
+        stockfish/src/types.h
+        stockfish/src/uci.cpp
+        stockfish/src/uci.h
+        stockfish/src/ucioption.cpp
+        stockfish/src/incbin/incbin.h
+        stockfish/src/syzygy/tbprobe.cpp
+        stockfish/src/syzygy/tbprobe.h
+        stockfish/src/nnue/evaluate_nnue.cpp
+        stockfish/src/nnue/evaluate_nnue.h
+        stockfish/src/nnue/nnue_accumulator.h
+        stockfish/src/nnue/nnue_architecture.h
+        stockfish/src/nnue/nnue_common.h
+        stockfish/src/nnue/nnue_feature_transformer.h
+        stockfish/src/nnue/architectures/halfkp_256x2-32-32.h
+        stockfish/src/nnue/features/feature_set.h
+        stockfish/src/nnue/features/features_common.h
+        stockfish/src/nnue/features/half_kp.cpp
+        stockfish/src/nnue/features/half_kp.h
+        stockfish/src/nnue/features/index_list.h
+        stockfish/src/nnue/layers/affine_transform.h
+        stockfish/src/nnue/layers/clipped_relu.h
+        stockfish/src/nnue/layers/input_slice.h
+        stockfish_bot.cc
+        stockfish_bot.h
+        )
+
+target_include_directories (stockfish PUBLIC stockfish/src)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -pthread -DNNUE_EMBEDDING_OFF")
```

Comment:
These seem like important compilation flags. Can you put this on top of the file? 
Is there a document which describes compilation flags that can be passed to Stockfish? Please add a link to them here, and possibly list the most important ones directly in the comment, with a short explanation. Thanks!